.pre-commit-config.yaml

```diff
@@ -1,10 +1,10 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: 19.10b0
+    rev: 20.8b1
```

Comment:
can you please change the items in this file back? These hooks should work as they are once you do `poetry install` in the cloned repository